src/components/transaction/TransactionDetails/DetailsByType/AssetTransferDetails/AssetTransferDetails.tsx

```diff
@@ -13,6 +13,13 @@ export default class AssetTransferDetails extends React.Component<Props, any> {
         const { tx } = this.props;
         const transaction = tx as TransferAssetTransactionDoc;
         return [
+            <Row key="metadata">
+                <Col md="3">Raw metadata</Col>
```

Comment:
Let's just make it `Metadata`.

Reply:
@joojis could you review my PR again? I've changed my commit with your review.